config/default/kustomization.yaml

```diff
@@ -19,6 +19,7 @@ bases:
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
 # crd/kustomization.yaml
 - ../webhook
+- ../ports
```

Comment:
Apparently even with this change, it's still possible to run into this error during deployment: 

```
error: resource mapping not found for name: "nnf-port-manager" namespace: "nnf-system" from "STDIN": no matches for kind "NnfPortManager" in version "nnf.cray.hpe.com/v1alpha1"
ensure CRDs are installed first
```

Reply:
Yes.  It has to be its own 'kubectl apply', so you can put a wait/watch between them to check for the CRD being ready.

Reply:
Thanks @roehrich-hpe. I started down this path and it seems to be working. Update incoming.

Reply:
Fixed.